test/e2e/set/scylladbmonitoring/scylladbmonitoring.go

```diff
@@ -188,12 +244,12 @@ func awaitScyllaDBMonitoringRollout(ctx context.Context, f *framework.Framework,
 func verifyManagedPrometheus(ctx context.Context, f *framework.Framework, sm *scyllav1alpha1.ScyllaDBMonitoring) {
 	g.GinkgoHelper()
 
-	// We need to retry the prometheus and grafana assertion for several reasons, some of them are:
-	//  - ingress exposure is asynchronous and some controllers don't report back status to wait for
-	//  - prometheus configuration is asynchronous without any acknowledgement
-	//  - grafana configuration is asynchronous without any acknowledgement
-	// Some of these may be fixable by manually verifying it in the operator sync loop so it can also be
-	// consumed by clients, but it's a bigger effort.
```

Comment:
Why is this being removed? I don't see any code changes that would explain it (it's still polling), but I might have lost some context.